google-http-client/src/main/java/com/google/api/client/util/FieldInfo.java

```diff
@@ -134,12 +134,20 @@ public static FieldInfo of(Field field) {
   }
 
   /** Creates list of setter methods for a field only in declaring class. */
-  private Method[] settersMethodForField(Field field) {
+  private Method[] settersMethodForField(final Field field) {
     List<Method> methods = new ArrayList<>();
+    String fieldSetter = "set" + Ascii.toUpperCase(field.getName().substring(0, 1));
```

Comment:
This approach works for this issue, is it possible to have a scenario that the field name is `utcTime`, and the setter is `setUTCTime`? We may have to look at how GenericJson is implemented to confirm this. So I think it would be safer to split the `method.getName()` by `set` and compare the name with `field.getName()`.

Reply:
The default setter for `utcTime` would be `setUtcTime()`. So, I think this logic would work. The default setter would take precedence over `setUTCTime()`.